android/assets/jsons/translations/German.properties

```diff
@@ -1695,6 +1695,7 @@ Dogfighting III = Kurvenkampf III
 Choose name for [unitName] = Wähle Namen für [unitName]
 [unitFilter] units gain the [promotion] promotion = [unitFilter] Einheiten erhalten die [promotion] Beförderung
 Requires = Benötigt
+- Path to [promotion] is ambiguous = (Der Weg zu [promotion] ist noch nicht klar!)
```

Comment:
This doesn't make sense to me. If you want it in brackets then bracket it, if you want it with a separator separate it. Adding the meta-formatting to the translation line is a surefire way to cause confution